kafka_utils/kafka_consumer_manager/commands/offsets_for_timestamp.py

```diff
@@ -83,6 +83,19 @@ def run(cls, args, cluster_config):
 
     @classmethod
     def print_offsets(cls, partition_to_offset, orig_timestamp):
+        milliseconds_thresold = 999999999999
```

Comment:
Can we add a comment here explaining why we have the threshold? It might not be clear to those reading

Reply:
Just added a comment. Can you please have a look? I feel like the comment is too long

Reply:
it's fine!